starter-code/scripts/articleView.js

```diff
@@ -36,6 +40,17 @@ articleView.handleCategoryFilter = function() {
   /* TODO: Just like we do for #author-filter above, we should also handle
   change events on the #category-filter element. Be sure to reset the
   #author-filter while you're at it! */
+  $('#category-filter').on('change', function() {
+    if ($(this).val()) {
+      $('article').hide();
+      $('article[data-category="' + $(this).val() + '"]').fadeIn('slow');
+      } else {
+    /* Otherwise, we should:
+        1. Show all the articles except the template */
+        $("article").not(".template").fadeIn('slow');
+    }
+    $('#category-filter').val('');
```

Comment:
So take a closer look at this. This isn't exactly what you want. HINT: look at the directions, what filter are you supposed to be targeting.